libs/domains/organizations/feature/src/lib/organization-overview/section-changelog/section-changelog.tsx

```diff
@@ -1,9 +1,10 @@
+import { useSuspenseQuery } from '@tanstack/react-query'
 import { Heading, Icon, Section } from '@qovery/shared/ui'
 import { dateFullFormat } from '@qovery/shared/util-dates'
-import { useChangelogs } from '@qovery/shared/webflow/feature'
+import { changelogsQuery } from '@qovery/shared/webflow/feature'
 
 export function SectionChangelog() {
-  const { data: changelogs = [] } = useChangelogs()
+  const { data: changelogs = [] } = useSuspenseQuery(changelogsQuery)
```

Comment:
What do you think to keep `useChangelogs` and replace `useSuspenseQuery` inside the hook? We can have it for all requests